### DISCUSSION TITLE
Disable "more lines", "fewer lines", etc... notifications

### DISCUSSION THREAD

**Discussion by academo** [LazyVim/LazyVim#2223] [General]
I am trying to find what is the plugin or configuration that keeps showing these notifications constantly. 

Do you know which plugin is this and how to disable these constant notifications?

Thanks in advance

**Comment by dpetka2001** [comment]
The plugin is `noice.nvim` and you would need to do something similar to this [recipe](https://github.com/folke/noice.nvim/wiki/Configuration-Recipes#hide-written-messages) to hide these messages.

**Reply by academo** [reply]
That's what I've been doing, filtering out these messages but there are too many notifications, for example, yanked text, saved and many other little things that confirm things I know I did and are a bit distracting.

Is there a general setting to disable those little notifications or should I filter them out one by one?

**Reply by dpetka2001** [reply]
You could try 
```lua
{
    "folke/noice.nvim",
    opts = function(_, opts)
      table.insert(opts.routes, {
        filter = {
          event = "msg_show",
          kind = "echo", -- or `kind = ""`
        },
        opts = {
          skip = true,
        },
      })
    end,
  },
```
and experiment with `kind = "echo"` or `kind = ""` to see what works for you. This way you might be filtering more than what you want though.

**Reply by academo** [reply]
Thanks for the tip! I'll give it a try :D

**Reply by jyuan0** [reply]
To add to the previous answer, an alternative:
```lua
        {
          filter = {
            event = "msg_show",
            any = {
              { find = "%d fewer lines" },
              { find = "%d more lines" },
              { find = "%d lines <ed %d time[s]?" },
              { find = "%d lines >ed %d time[s]?" },
              { find = "%d lines yanked" },
            },
          },
          view = "mini",
        },
```
I was worried about silencing more than intended, so I used `find` and changed to the `mini` view (lower right corner).

**Reply by academo** [reply]
Since you are being so helpful. would you mind telling me how to disable that notification when I start the editor? I tried what is mentioned here https://github.com/LazyVim/LazyVim/discussions/1479

and I have this in my config


```lua
    {
      "LazyVim/LazyVim",
      import = "lazyvim.plugins",
      opts = {
        checker = {
          notify = false,
        },
      },
    },
```
I keep seeing the notification on every start.

**Reply by dpetka2001** [reply]
This should go into your `/lua/config/lazy.lua` inside `lazy`'s setup. See [here](https://github.com/dpetka2001/dotfiles/blob/1a6165d7246de24fc8d751097df488f685f7e250/dot_config/nvim/lua/config/lazy.lua#L46-L49). It's an option for `lazy.nvim` the package manager, not LazyVim the plugin.

**Reply by Geobert** [reply]
> To add to the previous answer, an alternative:
> 
> ```lua
>         {
>           filter = {
>             event = "msg_show",
>             any = {
>               { find = "%d fewer lines" },
>               { find = "%d more lines" },
>               { find = "%d lines <ed %d time[s]?" },
>               { find = "%d lines >ed %d time[s]?" },
>               { find = "%d lines yanked" },
>             },
>           },
>           view = "mini",
>         },
> ```
> 
> I was worried about silencing more than intended, so I used `find` and changed to the `mini` view (lower right corner).

I don’t know what’s wrong with my routing but it doesn’t work :-/

https://codeberg.org/geobert/nvim-config/src/branch/main/lua/plugins/noice.lua

**Reply by folke** [reply]
@Geobert this is a discussion forum for LazyVim, not kickstart.
However, you are loading noice by doing that telescope noice thing, so that wouldn't work.

**Reply by Geobert** [reply]
Thanks! Sorry for the confusion, I’m new to nvim’s ecosystem :)

**Reply by Geobert** [reply]
@folke I commented out the telescope line but no luck :-/
